protected/humhub/modules/user/models/Group.php

```diff
@@ -92,11 +94,25 @@ public function attributeLabels()
     }
 
     /**
-     * @return null|Space
+     * @inheritdoc
+     */
+    public function attributeHints()
+    {
+        return [
+            'reassign-all-help' => Yii::t('AdminModule.settings','Reassign default spaces to all users'),
```

Comment:
Please do not work with attributeHints here. This link is not a attribute of the group model.

Reply:
Done!